### PR TITLE
add some additional analytics metrics

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: awx/main/analytics/collectors.py [hunk 1]
```diff
@@ -33,6 +35,12 @@ def something(since):
 def config(since):
     license_info = get_license(show_key=False)
     return {
+        'platform': {
+            'system': platform.system(),
+            'dist': platform.dist(),
+            'release': platform.release(),
+            'kubernetes': 'KUBERNETES_SERVICE_PORT' in os.environ,
```
Comment (ryanpetrello) [inline awx/main/analytics/collectors.py]: @shanemcd @matburt do you think this is a reasonable way to detect k8s?  These presence of these env vars are about the only reliable mechanism *I* know of.

I don't know of any way to determine "kubernetes_type (OCP, EKS, AKS, GKE, )" _inside_ the container.

Comment (ryanpetrello) [reply awx/main/analytics/collectors.py]: image

@benthomasson you have any feedback on this?

Comment (ryanpetrello) [reply awx/main/analytics/collectors.py]: image

Comment (ryanpetrello) [reply awx/main/analytics/collectors.py]: @benthomasson `platform.type` could be `["traditional", "openshift", "k8s"]`

Comment (benthomasson) [reply awx/main/analytics/collectors.py]: This is useful information to capture. Thanks!

**Review by matburt — APPROVED** [review]

**Review by rooftopcellist — APPROVED** [review]

**Comment by ryanpetrello** [comment]
regate

**Pull request merged**